packages/design-system/src/elements/toniq-input/toniq-input.element.ts

```diff
@@ -288,6 +295,7 @@ export const ToniqInput = defineToniqElement<{
                     })}
                     placeholder=${inputs.placeholder}
                 />
+                <div class="suffix">${inputs.suffix}</div>
```

Comment:
Let's only render this if the suffix actually exists:
```suggestion
                ${renderIf(!!inputs.suffix, html`<div class="suffix">${inputs.suffix}</div>`)}
```